admin_queries/generate_and_enroll_users.sql

```diff
@@ -0,0 +1 @@
+CALL users_randomly_generate($count::int, $course_instance_id);
```

Comment:
```suggestion
SELECT user_id, uid, uin, name, email
FROM users_randomly_generate($count, $course_instance_id);
```
Can we make this return information about the generated users like the above? (I know this will require changing the procedure to a function that returns the newly-inserted user rows.)

Also, I think we don't need the `::int` typecast?

Reply:
I guess I didn't have a reason to do anything with the output, and I couldn't make a function not have an output, so I switched to a procedure. What use case would you have for seeing the users?

Reply:
Two things: (1) it looks weird when there is no output, and it's unclear whether anything really happened, and (2) I wanted to login as one of the new users, but now I had to go off and query the DB to find their UID, so it would have been much easier to just know a valid UID right there.

Generally it seems like a trivial addition to return the users list?

Reply:
I added this.